examples/job/run-simulations-and-extract-properties.py

```diff
@@ -286,3 +290,9 @@
 html = dataframe_to_html(df)
 html
 
+
+# In[ ]:
+
+
+
+
```

Comment:
There might be a way to auto-delete trailing lines in pycharm, is there?

Reply:
It's possible to do that, but the root cause was having an extra cell in the corresponding notebook that was getting saved as blank space to the .py files